docs/UserGuide.adoc

```diff
@@ -49,287 +49,289 @@ This app will not work with earlier versions of Java 8.
 
-Examples:
+*Example:*
 
-* `complete 3`
-* `ct 3`
+* `history`
 
-=== Creating an issue on Github: `+issue` OR `ci`
+==== Refreshing the content : `refresh` OR `rf` `[coming in v2.0]`
 
-Creates an issue on github in the team repo. +
-*Format:* `+issue ti/TITLE [a/ASSIGNEES]... [ms/MILESTONE] [b/BODY] [l/LABELS]...` +
-OR +
-`ci ti/TITLE [a/ASSIGNEES]... [ms/MILESTONE] [b/BODY] [l/LABELS]...`
-
-[TIP]
-An issue has only 'title' field as compulsory. Rest are all optional. +
-An issue can have more than one assignees and labels.
+You can refresh the program to update it to the latest content. +
+*Format:* `refresh`
 
 *Examples:*
 
-* `+issue ti/complete issue a/johndoe ms/v1.1 b/CS2103T is a software engineering module l/type.task l/CS2103T`
-* `+issue ti/TestIssue`
-* `ci ti/Issue with only body b/test body l/type.test`
-
-=== Deleting a teammate : `delete` OR `d`
-
-Deletes the specified teammate from the ProgressChecker. +
-*Format:* `delete INDEX` OR `d INDEX`
+* `refresh`
+* `rf`
 
-****
-*Instructions:*
+==== Clearing all entries : `clear` OR `c`
 
-* Deletes the teammate at the specified `INDEX`.
-* The index refers to the index number shown in the most recent listing.
-* The index *must be a positive integer* 1, 2, 3, ...
-****
+You can clear all information and data inside the ProgressChecker if you want to restore the app. +
+*Format:* `clear` OR `c`
 
 *Examples:*
 
-* `list` +
-`delete 2` +
-Deletes the 2nd teammate in the ProgressChecker.
-* `find Betsy` +
-`d 1` +
-Deletes the 1st teammate in the results of the `find` command.
+* `clear`
+* `c`
 
-=== Editing a teammate : `edit` OR `ed`
+==== Exiting the program : `exit` OR `e`
 
-Edits an existing teammate in the ProgressChecker. +
-*Format:* `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...` +
-OR +
-`ed INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]...`
+You can exits the program when you are done with the work. +
+*Format:* `exit` OR `e`
 
-****
-*Instructions:*
+*Examples:*
 
-* Edits the teammate at the specified `INDEX`. The index refers to the index number shown in the last teammate listing. The index *must be a positive integer* 1, 2, 3, ...
-* At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
-* When editing tags, the existing tags of the teammate will be removed i.e adding of tags is not cumulative.
-* You can remove all the teammate's tags by typing `t/` without specifying any tags after it.
-****
+* `exit`
+* `e`
 
-*Examples:*
+=== Exercise
 
-* `edit 1 p/91234567 e/johndoe@example.com` +
-Edits the phone number and email address of the 1st teammate to be `91234567` and `johndoe@example.com` respectively.
-* `edit 2 n/Betsy Crower t/` +
-Edits the name of the 2nd teammate to be `Betsy Crower` and clears all existing tags.
+// tag::view[]
+==== Viewing a different tab : `view` OR `v`
```

Comment:
I think the view command falls under general usage and not under Exercise (although there will be some overlapping) because it is a command used to navigate all tabs in general. :)